docs/compass/04-01-installation.md

```diff
@@ -0,0 +1,78 @@
+# Enable Compass in Kyma
+
+To enable Compass in Kyma, follow the instructions for the [custom component installation](https://kyma-project.io/docs/master/root/kyma#configuration-custom-component-installation) and enable the `compass` and `compass-runtime-agent` modules. You can also [install Kyma on a cluster](https://kyma-project.io/docs/master/root/kyma#installation-install-kyma-on-a-cluster) with the ready-to-use configurations for different modes. There are two modes in which you can enable Compass in Kyma: default Kyma installation and Compass as a Central Management Plane.
+
+## Default Kyma installation
+
+![Kyma mode1](./assets/kyma-mode1.svg)
+
+This is a single-tenant mode which provides the complete cluster Kyma installation with all components, including Compass and the Runtime Agent. In this mode, Runtime Agent is already connected to Compass and they both work in a single-tenant mode as well. Using this mode, you can register external Applications in Kyma. To enable it, follow the cluster Kyma installation using the [`installer-cr-cluster-with-compass.yaml.tpl`](https://github.com/kyma-project/kyma/blob/master/installation/resources/installer-cr-cluster-with-compass.yaml.tpl) configuration file. Before you start the installation, apply the following ConfigMap which enables the [API Packages](https://github.com/kyma-incubator/compass/blob/master/docs/compass/03-packages-api.md) functionality:
+
+```yaml
+cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: api-packages-override
+  namespace: kyma-installer
+  labels:
+    installer: overrides
+    kyma-project.io/installation: ""
+data:
+  global.enableAPIPackages: "true"
```

Comment:
```suggestion
  global.disableLegacyConnectivity: "true"
```
This is outdated. The flag name has changed. See [this](https://github.com/kyma-project/kyma/pull/7876/files) PR.